src/test/java/net/syscon/elite/repository/jpa/repository/OffenderLanguageRepositoryTest.java

```diff
@@ -0,0 +1,39 @@
+package net.syscon.elite.repository.jpa.repository;
+
+import net.syscon.elite.repository.jpa.model.LanguageReferenceCode;
+import net.syscon.elite.repository.jpa.model.OffenderLanguage;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.ActiveProfiles;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase.Replace.NONE;
+
+@ActiveProfiles("test")
+@DataJpaTest
+@AutoConfigureTestDatabase(replace = NONE)
+public class OffenderLanguageRepositoryTest {
+
+    @Autowired
+    private OffenderLanguageRepository repository;
+
+    @Test
+    public void testGetLanguages() {
```

Comment:
Just in case you didn't know no need for public methods with Junit 5.  Personally I am not sure what benefit this adds...